apps/ledger-live-desktop/src/renderer/Default.tsx

```diff
@@ -333,6 +333,11 @@ export default function Default() {
                                 render={withSuspense(PlatformCatalog)}
                                 exact
                               />
+                              <Route
+                                path="/platform?category=:category"
+                                render={withSuspense(PlatformCatalog)}
+                                exact
+                              />
```

Comment:
I don't think you want the exact here
Also and more importantly I don't think you need a new route for a param that should already be accessible in the route above